db/schemas/schemas.py

```diff
@@ -15,3 +15,14 @@ class Meta:
 
     workflow_status = Enum(Status)
     language = Enum(Language)
+
+
+class AssessmentRecordMetadataWithJson(SQLAlchemyAutoSchema):
+    """AssessmentRecordMetadata The marshmallow class used to turn SQLAlchemy
+    rows into json for return in http responses."""
+
+    class Meta:
+        model = AssessmentRecord
+
+    workflow_status = Enum(Status)
+    language = Enum(Language)
```

Comment:
⛏️ Instead of this we can alter the existing Schema and remove the exclude. This will mean that the we will need to add some more `defer`s to the current queries.

Reply:
Have changed this here: https://github.com/communitiesuk/funding-service-design-assessment-store/commit/ff4aa11dbf05cb6f41bf0779f4efccc36d5a3c6d#diff-62e7f04d26309dfe4fe37ef06da26af5748ac7ed1346898667d2cffadc2b30d3

> This will mean that the we will need to add some more defers to the current queries.

Do you know where specifically? I don't see any others (at least on this branch) -- maybe something for after https://github.com/communitiesuk/funding-service-design-assessment-store/pull/46 is merged?